applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -112,14 +112,7 @@ CoulombYieldSurface::CoulombYieldSurface(const Properties& rMaterialProperties)
         mMaterialProperties[GEO_COULOMB_HARDENING_TYPE] = "None";
     }
 
-    if (!mMaterialProperties.Has(GEO_COULOMB_HARDENING_MAX_ITERATIONS)) {
-        mMaterialProperties[GEO_COULOMB_HARDENING_MAX_ITERATIONS] = 100;
-    }
-
-    if (!mMaterialProperties.Has(GEO_COULOMB_HARDENING_CONVERGENCE_TOLERANCE)) {
-        mMaterialProperties[GEO_COULOMB_HARDENING_CONVERGENCE_TOLERANCE] = 1.0e-8;
-    }
-
+    this->CheckMaterialProperties();
```

Comment:
No `this` pointer required here, so please leave it out.
```suggestion
    CheckMaterialProperties();
```